data/World/America/USA/utah-turn.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+        "name": "TURN Utah",
+        "description": "The Utah Reference Network (TURN)",
+        "urls": [
+            "http://165.239.144.5:2101",
+            "http://turngps.utah.gov:2101"
+        ],
+        "reference": {
+            "url": "https://gis.utah.gov/documentation/turn/connecting/"
+        },
+        "last_update": "2025-01-27",
+        "streams": [
+            {
+                "filter": "all",
```

Comment:
Shouldn't the keyword `filter` just be omitted? If you're filtering all you'll end up with nothing 😉

Reply:
We had that discussion and Javier wanted to have something explicit.